lib/periodicChecks.js

```diff
@@ -0,0 +1,198 @@
+      await mergeConflictModule.checkMergeConflictsInPullRequest(
+        context,
+        pullRequestData
+      );
+    } else if (hasPendingReviews(pullRequestData)) {
+      console.log('PULL REQUEST HAS PENDING REVIEWS...');
+      const reviewers = pullRequest.requested_reviewers.map(
+        (reviewer) => reviewer.login
+      );
+      await pingAndAssignReviewers(context, pullRequestData, reviewers);
+    } else if (hasChangesRequested) {
+      console.log('PULL REQUEST HAS CHANGES REQUESTED...');
+      const review = await getLastReviewOfType(
+        context,
+        CHANGES_REQUESTED,
+        pullRequestData.number
+      );
+
+      pullRequestReviewModule.handleChangesRequested(
+        context,
+        pullRequestData,
+        review
+      );
+    } else if (hasBeenApproved) {
+      console.log('PULL REQUEST HAS BEEN APPROVED...');
+      await pullRequestReviewModule.handleApprovalByAllReviewers(
+        context,
+        pullRequestData
+      );
+    } else {
+      console.log('CONFUSED STATE, ASK FOR HELP FROM WELFARE TEAM...');
+      context.github.issues.createComment(
+        context.repo({
+          issue_number: pullRequestData.number,
+          body:
+            'Hi @' +
+            teamLeads.onboardingTeam +
+            ', @' +
+            oppiaMaintainers +
+            ' -- flagging this PR since it needs some assistance. PTAL, ' +
+            'thanks!',
+        })
+      );
+
+      context.github.issues.addAssignees(
+        context.repo({
+          issue_number: pullRequestData.number,
+          assignees: [teamLeads.onboardingTeam],
+        })
+      );
+    }
+  });
+};
+
+/**
+ * This function ensures that all issue are linked to a project.
+ * 1. Fetch all open issues.
+ * 2. Fetch all project cards.
+ * 3. Filter open issues by the ones that have the triage label.
+ * 4. Filter open issues by the ones that have a project card.
+ * 5. Get Remaining issues
+ * 6. Add todo triage label.
+ * 7. Ping core maintainers
+ * @param {import('probot').Context} context
+ */
+const ensureAllIssuesHaveProjects = async (context) => {
+  const allOpenIssues = await getAllOpenIssues(context);
+  const allProjectCards = await getAllProjectCards(context);
+
+  const issuesWithoutProjects = allOpenIssues.filter((issue) => {
+    const issueBelongsToProject = allProjectCards.some((card) => {
+      return card.content_url.endsWith(issue.number);
+    });
+    return !issueBelongsToProject;
+  });
+
+  issuesWithoutProjects.forEach((issue) => {
+    const labels = issue.labels.map((label) => label.name);
+    if (labels.includes(TODO_TRIAGE_LABEL)) {
+      // If label has already been added we do nothing since we have already
+      // pinged the core maintainers.
+      return;
+    }
+    // Add Todo triage label.
+    context.github.issues.addLabels(
+      context.repo({
+        issue_number: issue.number,
+        labels: [TODO_TRIAGE_LABEL],
+      })
+    );
```

Comment:
We should not do any of this - just check for project and ping. The label is for project leads to assign a priority to the issue.